sdk/healthinsights/health-insights-cancerprofiling-rest/recordings/browsers/my_test/recording_cancer_profiling_test.json

```diff
@@ -0,0 +1,370 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://endpoint/healthinsights/oncophenotype/jobs?api-version=2023-03-01-preview",
```

Comment:
Could you follow the guide to migrate the recordings to asset repo? Simply speaking you can just run below command:

```
$ npx dev-tool test-proxy migrate --initial-push
```

https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/recorder/ASSET_SYNC_MIGRATION.md#migration